features/polyExplorer/src/screens/stories/digitalGiantsStory.jsx

```diff
@@ -1,12 +1,17 @@
 import React, { useContext } from "react";
 
 import ClusterStory from "../../components/clusterStory/clusterStory.jsx";
+import GradientCircleList from "../../components/gradientCircleList/gradientCircleList.jsx";
 import { ExplorerContext } from "../../context/explorer-context.jsx";
 import i18n from "../../i18n.js";
-//import SectionTitle from "../../components/clusterStories/sectionTitle.jsx";
-//import Tab from "../../components/clusterStories/tab.jsx";
+import SectionTitle from "../../components/clusterStories/sectionTitle.jsx";
+import { Tabs, Tab } from "@polypoly-eu/poly-look";
+import { createJurisdictionLinks } from "./story-utils";
+import { PolyChart } from "@polypoly-eu/poly-look";
 
 const i18nHeader = "clusterDigitalGiantsStory";
+const i18nHeaderCommon = "clusterStoryCommon";
+const primaryColor = "#f95f5a";
```

Comment:
There is a variable for `--color-primary-messenger-story` in the "main" branch (in styles.css). It might be a good idea to add also a variable for this color to keep consistency.

Reply:
I think rather have the color here, but you're right we need to do this in the other one as well

Reply:
Ok, let's do it after merging it with `main`. I'll write a ticket for this - I know it's a short task, but just to not forget about it :)